radicle-httpd/src/api/v1/projects.rs

```diff
@@ -467,3 +467,393 @@ fn stats(repo: &Repository, head: Oid) -> Result<Stats, Error> {
         contributors: contributors.len(),
     })
 }
+
+#[cfg(test)]
+mod routes {
+    use crate::api::v1::utils::{get_response, json, HEAD};
+
+    #[tokio::test]
+    async fn test_projects_root() {
+        let body = get_response("/v1/projects").await;
+
+        assert_eq!(
+            body,
+            json!([{
+               "name":"hello-world",
+               "description":"Rad repository for tests",
+               "defaultBranch":"master",
+               "head":HEAD,
+               "patches":0,
+               "issues":1,
+               "id":"rad:z4FucBZHZMCsxTyQE1dfE2YR59Qbp"
```

Comment:
Can we format all JSON the same, with a space between the `:` and value?, ie. `{"foo": 1}`

Reply:
fixed